lambdas/get-users-id-request/src/testData.ts

```diff
@@ -0,0 +1,63 @@
+import { AddressData, RequestData } from '../../../common/help-on-spot-models/dist/models/RestModels'
```

Comment:
At some point we should place all that testdata stuff in the commons package instead of copying it form one lambda to another. But this is a good start